docker/build.sh

```diff
@@ -35,7 +36,7 @@ cd "$CURRENT"
 
 # build
 echo "Building docker image"
-docker-compose build
+docker-compose build --pull --no-cache
```

Comment:
This pulls the base image and also disables caching (which would cache the apt-get upgrade command).